Rocket.Chat.iOS.xcodeproj/project.pbxproj

```diff
@@ -564,7 +564,7 @@
 				CODE_SIGN_IDENTITY = "iPhone Developer";
 				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
 				INFOPLIST_FILE = Rocket.Chat.iOS/Info.plist;
-				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 9.0;
```

Comment:
@gsaslis I don't think so... the same devices that support iOS 8 will support iOS 9. Also, iOS 9 have a lot of cool features that we can use by targeting it.